tgui/packages/tgui/interfaces/IntegratedCircuit/Connections.jsx

```diff
@@ -19,6 +19,7 @@ export const Connections = (props) => {
         position: 'absolute',
         pointerEvents: 'none',
         zIndex: -1,
+        overflow: 'visible',
```

Comment:
this is the default. are you sure its needed?

Reply:
Blindly taken from https://github.com/tgstation/tgstation/pull/89523

Reply:
suppose it doesn't hurt